tools/eval/README.md

```diff
@@ -14,6 +14,7 @@ python eval.py -m model_name -d dataset_name -dr dataset_root_dir
 
 Supported datasets:
 - [ImageNet](./datasets/imagenet.py)
+- [ICDAR](./datasets/icdar.py)
```

Comment:
I think it needs to be changed to [ICDAR]\(#icdar2003) here. That way we can jump to this section instead of the source code.
I have changed this part for  imagenet in #70.